docs/development/updating-components.md

```diff
@@ -136,3 +136,22 @@ helm fetch --untar --untardir ./ bitnami/external-dns
 ```
 
 - Chart location: https://github.com/bitnami/charts/tree/master/bitnami/external-dns.
+
+## aws-ebs-csi-driver
+
+Run the following commands in the root of this repository:
+
+```bash
+cd assets/charts/components
+rm -rf aws-ebs-csi-driver
+helm repo add aws-ebs-csi-driver https://kubernetes-sigs.github.io/aws-ebs-csi-driver
+helm repo update
+helm fetch --untar --untardir ./ aws-ebs-csi-driver/aws-ebs-csi-driver
+git checkout aws-ebs-csi-driver/templates/networkpolicy.yaml
+git checkout aws-ebs-csi-driver/templates/volumesnapshotclass.yaml
+git checkout aws-ebs-csi-driver/crds
+```
```

Comment:
Shoudn't this instructions be part of the update script whenever aws-ebs-csi-driver is updated, rather than having it a documentation and putting the task on user ?

Reply:
This is not a user documentation, but a developer documentation.

Reply:
never mind, didn't see the `docs/development/...` so its not for the user.